hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1544,7 +1544,7 @@ public String getConcatHandleClassName() {
   }
 
   public String getFileGroupReaderMergeHandleClassName() {
-    return getStringOrDefault(FILE_GROUP_READER_MERGE_HANDLE_CLASS_NAME);
+    return getStringOrDefault(COMPACT_MERGE_HANDLE_CLASS_NAME);
```

Comment:
we should always match the config name and getter methods. 
`getCompactMergeHandleClassName`